client/js/components/statement/segments/DpRecommendationModal.vue

```diff
@@ -7,62 +7,58 @@
   All rights reserved
 </license>
 
-<template>
-  <dp-modal
-    ref="recommendationModal"
-    content-classes="u-2-of-3">
-    <template>
-      <h3>{{ Translator.trans('segment.recommendation.insert.similar') }}</h3>
-      <div class="layout u-mb">
-        <div class="layout__item u-1-of-3">
-          <span class="display--block weight--bold">
-            {{ Translator.trans('segment.tags') }}
+<dp-modal
```

Comment:
thanks for clarifying we do not need a top-level `<template>` if the top-level component is a custom one.